writeengine/wrapper/CMakeLists.txt

```diff
@@ -41,6 +41,6 @@ add_library(writeengine SHARED ${writeengine_LIB_SRCS})
 
 add_dependencies(writeengine loggingcpp)
 
-target_link_libraries(writeengine ${NETSNMP_LIBRARIES})
```

Comment:
Should be a variable set in the top CMakeLists.txt but I suppose it is set in develop.

Reply:
No, it's not. IMHO bundling the libraries by toplevel varaibles can be useful, but should not be required. Arrow build support is made the way, the only thing one needs to do, to use, just to link it.